menu/reader.js

```diff
@@ -0,0 +1,41 @@
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Menu {
+    constructor(text) {
+        this.delivery = text['Delivery'];
+        this.instructions = text['Instructions']; // If instructions provided
+        this.dishes = text['Menu'];
+        this.modifiers = text['Modifiers'];
+    }
+}
+
+// Convert .yml file to yml text
+function readYmlFile(filename) {
```

Comment:
this would be an inappropriate function name, because it is irrelevant whether the file is yml